CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to the Rails Girls Summer of Code website
+
+We're always very happy about contributions to our website, so thank you for taking the time to contribute! Here you will find some guidelines about what is expected from contributors. Before getting started, please make sure you read and understand our [Code of Conduct](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CODE_OF_CONDUCT.md), and be aware that every contributor is expected to abide by it.
+
+#### Table of contents
+
+1. [Setting up and getting started](#setting-up-and-getting-started)
+2. [Reporting a bug](#reporting-a-bug)
+3. [Suggesting a new feature](#suggesting-a-new-feature)
+4. [Contributing to an existing issue](#contributing-to-an-existing-issue)
+5. [Additional info](#additional-info)
+
+
+
+
+## Setting up and getting started
+
+The Rails Girls Summer of Code website is built using [Jekyll](https://jekyllrb.com/). In order to set up and run the site locally for development, you'll need to have Ruby installed on your machine, create a GitHub account (if you don't already have one), and fork our repo.  
```

Comment:
I'd remove the "(if you don't already have one)" since it does not really add anything useful?